R/score_model_out.R

```diff
@@ -72,7 +78,11 @@
 #' )
 #' head(pmf_scores)
 #'
-#' @return forecast_quantile
+#' @return A data.table with scores
+#'
+#' @references
+#' Making and Evaluating Point Forecasts, Gneiting, Tilmann, 2011,
+#' Journal of the American Statistical Association.
```

Comment:
@bsweger was wondering if we preferred APA or Chicago and I went with Chicago. 

```suggestion
#' Gneiting, Tilmann. 2011. "Making and Evaluating Point Forecasts." Journal of the 
#' American Statistical Association 106 (494): 746–62. <doi: 10.1198/jasa.2011.r10138>.
```